test/api/unit/models/build.test.js

```diff
@@ -52,16 +52,44 @@ describe('Build model', () => {
   });
 
   describe('after create hook', () => {
+    it('should send a build new build message', (done) => {
+      const startedAt = new Date();
+      factory.build({ state: 'processing', startedAt })
+        .then((build) => {
+          expect(build.completedAt).to.be.null;
+          return build.updateJobStatus({ status: 'success' });
```

Comment:
If the subject of this test is the `afterCreate` hook, why do we need to call `updateJobStatus` here? Testing that functionality should probably be in a separate spec.

Reply:
i spent much time debugging this this morning ... even @apburnes joined in. we uncovered that it it was introduced as a delay ... i have replaced it with a different delay